commit d810c73396ec7931addf3e7f75b17daf4e184bf4
Author: Adam Rankin <adam.rankin@gmail.com>
Date:   Sun Mar 11 22:28:18 2018 -0400

    Update precomp.hpp
    
    Enabling build of visualization module when using VTK 9 (current HEAD of d5bbb9e99bbc6d11d2196c48bfd8f33508554551)

diff --git a/modules/viz/src/precomp.hpp b/modules/viz/src/precomp.hpp
index 369db191d3..dc5a5db1ea 100644
--- a/modules/viz/src/precomp.hpp
+++ b/modules/viz/src/precomp.hpp
@@ -104,6 +104,7 @@
 #include <vtkMath.h>
 #include <vtkExtractEdges.h>
 #include <vtkFrustumSource.h>
+#include <vtkTexture.h>
 #include <vtkTextureMapToPlane.h>
 #include <vtkPolyDataNormals.h>
 #include <vtkAlgorithmOutput.h>
